src/components/AssetManagementPanel/AssetManagementApplication/index.tsx

```diff
@@ -18,6 +18,7 @@ export const AssetManagementApplication = ({
   titleEscrow,
 }: AssetManagementApplicationProps) => {
   const [assetManagementAction, setAssetManagementAction] = useState(AssetManagementActions.None);
+  const [holderErrorState, setHolderErrorState] = useState("");
```

Comment:
Dont need this, you have the `changeHolderState` already

Reply:
understand that, but if i click cancel or back, the changeHolderState is still "ERROR", which will show the error msg when click on transfer holder again.